.github/workflows/ci.yml

```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4.2.2
-      - name: Set up Python 3.10
+      - name: Set up Python 3.13
```

Comment:
I think we should keep using the oldest supported Python version for the linting job.